resources/session02/homework/http_server.py

```diff
@@ -35,7 +40,17 @@ def parse_request(request):
 
 def resolve_uri(uri):
     """This method should return appropriate content and a mime type"""
-    return b"still broken", b"text/plain"
+    path = pathlib.Path('webroot{}'.format(uri))
```

Comment:
It looks to me like it's more canonical with pathlib to build a path for 'webroot' and then another for the relative form of uri (`pathlib.Path(uri.lstrip('/'))`), and then concatenate them with '/':

```python
target_path = webroot_path / relative_uri
```

But this form you use here is equally workable.  A possible advantage of the former approach would be that you could check that the final path was actually a subdirectory of the webroot path.  We wouldn't want to allow requesting code to climb out of the web server's root directory using `..`, would we?